app/assets/src/components/common/DetailsSidebar/GeneDetailsMode/GeneDetailsMode.jsx

```diff
@@ -0,0 +1,430 @@
+    if (loading) {
+      return <div className={cs.loadingMsg}>Loading...</div>;
+    }
+    return (
+      <div className={cs.title}>
+        {ontology.label !== "---" ? ontology.label : geneName}
+      </div>
+    );
+  }
+
+  renderOntology() {
+    const { ontology, collapseOntology } = this.state;
+    return (
+      <div>
+        <div className={cs.text}>
+          <div className={cs.textInner}>
+            {CARD_SYNONYMS}: <em>{ontology.synonyms}</em>
+          </div>
+        </div>
+        <div className={cs.subtitle}>Description</div>
+        <div className={cs.text}>
+          <div className={cs.textInner}>{ontology.description}</div>
+        </div>
+        {!collapseOntology && (
+          <div>
+            <div className={cs.subtitle}>{CARD_FAMILY}</div>
+            <div className={cs.text}>
+              <div className={cs.textInner}>{ontology.geneFamily}</div>
+            </div>
+            <div className={cs.subtitle}>{CARD_CLASS}</div>
+            <div className={cs.text}>
+              <div className={cs.textInner}>{ontology.drugClass}</div>
+            </div>
+            <div className={cs.subtitle}>{CARD_MECHANISM}</div>
+            <div className={cs.text}>
+              <div className={cs.textInner}>{ontology.resistanceMechanism}</div>
+            </div>
+            <div className={cs.subtitle}>{CARD_PUBLICATIONS}</div>
+            <div className={cs.text}>{this.renderPublications()}</div>
+          </div>
+        )}
+        {collapseOntology && (
+          <div className={cs.expandLink} onClick={this.expandOntology}>
+            Show More
+          </div>
+        )}
+        <div className={cs.text}>
+          <div className={cs.textInner}>{this.renderCARDLicense()}</div>
+        </div>
+      </div>
+    );
+  }
+
+  renderPublications() {
+    const { ontology } = this.state;
+    return ontology.publications.map(publication => {
+      const citation = /.*(?=(\(PMID))/.exec(publication)[0];
+      const pmidText = /(PMID)\s[0-9]*/.exec(publication)[0];
+      const pubmedId = pmidText.split(" ")[1];
+      return (
+        <div className={cs.textInner} key={pubmedId}>
+          {citation}
+          <div className={cs.link}>
+            (<a
+              href={URL_PUBMED + pubmedId}
+              className={cs.link}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              {pmidText}
+            </a>)
+          </div>
+        </div>
+      );
+    });
+  }
+
+  renderGeneContents() {
+    const { loading, cardEntryFound } = this.state;
+    if (loading) {
+      return;
+    }
+    return (
+      <div className={cs.geneContents}>
+        {cardEntryFound ? this.renderOntology() : this.renderNotFound()}
+        <div className={cs.subtitle}>Links</div>
+        <div className={cs.linksSection}>
+          <ul className={cs.linksList}>
+            {cardEntryFound && (
+              <li className={cs.link}>
```

Comment:
you could DRY this out by having a render function for the <li> blocks since they're all the same

Reply:
Done!